cmd/guacgql/cmd/ingest.go

```diff
@@ -741,7 +742,7 @@ func ingestCertifyBad(ctx context.Context, client graphql.Client) {
 	ingestCertifyBad := []struct {
 		name         string
 		pkg          *model.PkgInputSpec
-		pkgMatchType *model.MatchFlags
+		pkgMatchType model.MatchFlags
```

Comment:
i assume this change was because we needed batch to force the definition of something per index of the list, can we set a default value for this?

Reply:
Does it make sense to make `pkgMatchType` required field via the API?